### DISCUSSION TITLE
Question: How to change Organization Packages permissions via API?

### DISCUSSION THREAD

**Discussion by Timmmy-nlb** [community/community#120859] [Enterprise]
How can I change this via API?

Organization:
Packages permissions / Package creation
to "Internal"

![grafik](https://github.com/community/community/assets/52195132/d0837a67-0ac3-4a70-876e-b4cc32181593)

**Comment by queenofcorgis** [comment]
This is also not possible 😢. 

Packages inherits permissions by default and it appears that the only way to break that inheritance is via the UI.

**Reply by Timmmy-nlb** [reply]
Hallo and thank you for reply.
But sorry, this is a enterprise environment. In company's is no space for 'private' and extra right configuration.
Everybody must has to access to packages - from another enterprise organization too.
